packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -307,6 +307,18 @@ const ERRORS = {
     errno: 206,
     message: 'Can not create password, password already set',
   },
+  RECOVERY_PHONE_NUMBER_ALREADY_EXISTS: {
+    errno: 214,
+    message: 'Recovery phone number already exists',
+  },
+  RECOVERY_PHONE_NUMBER_DOES_NOT_EXIST: {
+    errno: 215,
+    message: 'Recovery phone number does not exist',
+  },
+  TEXT_MESSAGE_LIMIT_REACHED: {
+    errno: 216,
+    message: 'Text message limit reached',
+  },
```

Comment:
I think we also need to add: `SMS_SEND_RATE_LIMIT_EXCEEDED`

Reply:
Or rather, `TEXT_MESSAGE_LIMIT_REACHED` should be replaced with `MS_SEND_RATE_LIMIT_EXCEEDED`?

Reply:
Thanks, that was a copy paste error on my part. Good catch.